RELEASING.md

```diff
@@ -0,0 +1,22 @@
+# Releasing the .NET Functions Frameworks packages
+
+This is a largely automated process. Steps:
+
+- Decide on a new version number, following semantic versioning, but
+  don't change any files.
+- Document the new version in [docs/history.md]() (but don't commit)
+- Run `./create-release-commit.sh {new-version}` from the root directory, passing
```

Comment:
nit: incomplete, "passing..."

Reply:
Whoops, thanks. I ended up putting it in the quotes - will remove ", passing"